src/utils/index.ts

```
@@ -3,4 +3,12 @@
  * @returns 奇数のみを含む配列
 */
-export const odd = (numbers: number[]): number[] => numbers.filter(num => num % 2 !== 0);
\ No newline at end of file
+export const odd = (numbers: number[]): number[] => numbers.filter(num => num % 2 !== 0);
+
+export const even = (numbers: number[]): number[] => numbers.filter(num => num % 2 === 0);
+
+export const hogehoge = (number: number): number => {
+         if(isNaN(number)) return number
+         if(isNaN(number)) return number
+                           return number *2
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)